src/client/java/teammates/client/scripts/testdataconversion/DataStoreToSqlConverter.java

```diff
@@ -57,8 +57,7 @@ public class DataStoreToSqlConverter {
     private UuidGenerator accounRequestUuidGenerator = new UuidGenerator(initialAccountRequestNumber, uuidPrefix);
     private UuidGenerator sectionUuidGenerator = new UuidGenerator(initialSectionNumber, uuidPrefix);
     private UuidGenerator teamUuidGenerator = new UuidGenerator(initialTeamNumber, uuidPrefix);
-    private UuidGenerator deadlineExtensionUuidGenerator = new UuidGenerator(initialDeadlineExtensionNumber,
-            uuidPrefix);
+    private UuidGenerator deadlineExtensionUuidGenerator = new UuidGenerator(initialDeadlineExtensionNumber, uuidPrefix);
```

Comment:
Here are some of my formatter problems that we did not catch last time. This reverts the formatter change to be the same as on master